GeosCore/aerosol_mod.F90

```diff
@@ -475,28 +481,28 @@ SUBROUTINE AEROSOL_CONC( Input_Opt,  State_Chm, State_Diag, &
              IF ( IS_HMS ) THEN
 
                 !%%%%% Fullchem simulations: add contribution from HMS
-                SO4_NH4_NIT(I,J,L) = ( Spc(id_SO4)%Conc(I,J,L)     &
-                                   +   Spc(id_HMS)%Conc(I,J,L)     &
-                                   +   Spc(id_NH4)%Conc(I,J,L)     &
-                                   +   Spc(id_NIT)%Conc(I,J,L) )   &
+                SO4_NH4_NIT(I,J,L) = ( Spc(I,J,L,id_SO4)                     &
+                                   +   Spc(I,J,L,id_HMS)                     &
+                                   +   Spc(I,J,L,id_NH4)                     &
+                                   +   Spc(I,J,L,id_NIT) )                   &
```

Comment:
Hi Bob, I didn't add this dimension. It is originally in GCHPv13.4.0. 
I don't think removing this would affect implementing my parameterization. What I need essentially is to add one dimension to `Rad_dry` and `Rad_wet` ect. at around line 395 to include allow varying dry size.

Reply:
Maybe it would be easier for you to implement this if I create a new PR based on 14.3.0.